docs/getting-started-guides/coreos/coreos_multinode_cluster.md

```diff
@@ -59,9 +59,9 @@ aws ec2 run-instances \
 --user-data file://node.yaml
 ```
 
-### GCE
+### Google Compute Engine
```

Comment:
I left them out because in this document there are no additional references but i just followed the CoreOS link and they use GCE all over the place so appending the (GCE) is warranted.